src/Lykke.Logs/Loggers/LykkeAzureTable/LykkeAzureTableLoggerExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using JetBrains.Annotations;
+using Lykke.SettingsReader;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Lykke.Logs.Loggers.LykkeAzureTable
+{
+    [PublicAPI]
+    public static class LykkeAzureTableLoggerExtensions
+    {
```

Comment:
I would add methods similar to https://github.com/LykkeCity/Lykke.Logs/pull/46/files#diff-c3d187827adeaae101f87dd9173c7c9bR11
To support the following scenario
https://github.com/aspnet/Mvc/blob/edf4e8fd9e238409f77c34ebd66d8ed8be332962/samples/MvcSandbox/Startup.cs#L47